content/en/project/_index.md

```diff
@@ -10,17 +10,19 @@ aliases:
 ---
 ## What is the Medley Interlisp Project?
 
-The Medley Interlisp Project was created to revive Medley Interlisp.
+The Medley Interlisp Project was created to revive the "Interlisp experience" by creating an environment that could be used by (and might even be useful for) modern users. Starting from the 20-year-old sources for the not-quite-released Medley 3.5 version of Interlisp, we have made good progress adapting Medley to work in the current environment -- open source, GitHub integration, improving Common Lisp support, running on a wide variety of platforms -- but there is much more to be done.
```

Comment:
could be used -> can be used

in the current environment -> in today's environment

Is open source an activity or was it the impetus that has allowed the activity of reviving Medley Interlisp to go forward?